mautic/mautic
# [UI] Better publish badge

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependency:
- #13689 

This PR introduces an enhancement for the active/available labels.
Now, they are context-specific to show the right label for each resource.




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/365073f6-84b6-4af7-8cf9-87272d1af5c1)                                      | ![image](https://github.com/user-attachments/assets/042ed661-aad4-4389-840a-bf50574f7cd2) ![image](https://github.com/user-attachments/assets/36fe6709-952f-4b9d-9ad9-aeffc178b1ac) ![image](https://github.com/user-attachments/assets/30d5ac0f-b120-4b98-8c28-58d490045cac) ![image](https://github.com/user-attachments/assets/0549f0fb-e9c8-4299-a49c-4722fce4f46a)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new campaign and save it. Then, close the campaign and verify that the "Active/Inactive" label is visible.
